pkg/volume/plugins.go

```diff
@@ -34,6 +34,14 @@ import (
 	"k8s.io/kubernetes/pkg/util/mount"
 )
 
+const (
+	// Common parameter which can be specified in StorageClass to specify the desired FSType
+	// Provisioners SHOULD implement support for this if they are block device based
+	// Must be a filesystem type supported by the host operating system.
+	// Ex. "ext4", "xfs", "ntfs". Default value depends on the provisioner
+	VolumeParameterFSType = "fstype"
```

Comment:
As far as I remember, the parameter types that were already present were all case insensitive, so I decided to make the constant all lower case.